frontend/src/metabase-types/api/search.ts

```diff
@@ -99,10 +99,19 @@ export interface SearchResult<
 }
 
 export type SearchContext =
-  | "search-bar"
-  | "search-app"
+  | "available-models"
   | "command-palette"
-  | "entity-picker";
+  | "entity-picker"
+  | "metabot"
+  | "metrics-browser"
+  | "model-upload"
+  | "model-browser"
+  | "question-picker"
+  | "search-app"
+  | "search-bar"
+  | "skip-token" // never sent to the backend, used in a sentinel payload
+  | "strategy-editor"
```

Comment:
Not sure what this is, but related to `StrategyEditorForQuestionsAndDashboards.tsv`